File: src/IsbInterop/Accessory/ISystemInfo.vb
Namespace Accessory
  ''' <summary>
  ''' Информация о системе.
  ''' </summary>
  Public Interface ISystemInfo
    Inherits IIsbComObjectWrapper
    ''' <summary>
    ''' Версия клиента.
    ''' </summary>
    ReadOnly Property ClientVerson() As String
  End Interface
End NameSpace

File: src/IsbInterop/Accessory/ISystemInfo.cs
namespace IsbInterop.Accessory
{
  /// <summary>
  /// Информация о системе.
  /// </summary>
  public interface ISystemInfo : IBaseIsbObject
  {
    /// <summary>
    /// Версия клиента.
    /// </summary>
    string ClientVerson { get; }
  }
}
